### DISCUSSION TITLE
Where can I post a suspected error in the MicroPython documentation?

### DISCUSSION THREAD

**Discussion by kp2809** [micropython/micropython#12747] [ESP32]
I'm a newbie to MicroPython and this forum. Where can I post a suspected error in the MicroPython documentation?

**Comment by jimmo** [comment]
https://github.com/micropython/micropython/issues/new?assignees=&labels=documentation&projects=&template=documentation.md&title=docs%3A+

**Reply by jimmo** [reply]
(Linked from https://github.com/micropython/micropython/issues/new/choose)
